bitnami/harbor/templates/core/core-dpl.yaml

```diff
@@ -0,0 +1,148 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "harbor.core" . }}
+  labels:
+    {{ include "harbor.labels" . | nindent 4 }}
+    app.kubernetes.io/component: core
+spec:
+  replicas: {{ .Values.core.replicas }}
+  selector:
+    matchLabels:
+      {{ include "harbor.matchLabels" . | nindent 6 }}
```

Comment:
Use a single line:
```diff
-     matchLabels:
-       {{ include "harbor. matchLabels" . | nindent 6 }}
+     matchLabels: {{ include "harbor. matchLabels" . | nindent 6 }}
```
Apply this change to every template containing 'matchLabels'